docs/spelling_wordlist.txt

```diff
@@ -0,0 +1,7 @@
+IPv
```

Comment:
@abhinavsingh this could probably be moved to the glossary and linked as a term once it's out of the README.